docs/pages/includes/preset-roles-table.mdx

```diff
@@ -1,13 +1,13 @@
-| Role | Description |
-| --- | --- |
-| `access`| Allows access to cluster resources. |
-| `editor` | Allows editing of cluster configuration settings. |
-| `auditor`| Allows reading cluster events, audit logs, and playing back session records. |
-| `requester`| Enterprise-only role that allows a user to create Access Requests. |
-| `reviewer`| Enterprise-only role that allows review of Access Requests. |
-| `group-access`| Allows access to all user groups. | 
-| `device-admin`| Used to manage trusted devices. | 
-| `device-enroll`| Used to grant device enrollment powers to users. |
-| `require-trusted-device`| Requires trusted device access to resources. |
-| `terraform-provider`| Allows the Teleport Terraform provider to configure all of its supported Teleport resources. |
+| Role | Description | Enterprise-only |
+| --- | --- |  --- | --- |
```

Comment:
```suggestion
| --- | --- |  --- |
```
The current table isn't rendering. I'm pretty sure this will fix that, but we'll want to make sure before merging